pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -150,12 +158,12 @@ func lookupDomainStatus(ctx *zedmanagerContext, key string) *types.DomainStatus
 }
 
 func publishDomainConfig(ctx *zedmanagerContext,
-	status *types.DomainConfig) {
+	config *types.DomainConfig) {
```

Comment:
Nice catch and clean-up. A small thing, but calling a `DomainConfig` a `status` always is confusing.